README.md

```diff
@@ -34,6 +34,7 @@ On release, automated continuous integration tests run the pipeline on a full-si
    - Low complexity filtering
    - Host read removal
    - Run merging
+   - Adapter and quality trimming of Nanopore reads
```

Comment:
I don't think this is necessary as a separate point. You can combine it with the first subpoint (i.e., `- Adapter clipping and merging (short, and nanopore reads)

Reply:
Yes, I fixed now the README.md and updated the preprocessing subworkflow name.